cgi-bin/DW/Controller/Talk.pm

```diff
@@ -176,16 +179,39 @@ sub talkpost_do_handler {
     # For errors that aren't immediately fatal, collect them as we go and let
     # the user fix them all at once.
     my @errors;
+    my $form_auth_ok = 1;
```

Comment:
Might be a nit, but I always prefer these things to logically default to false, and then have a very explicit "this is when we turn it on" flow. Otherwise, we have to make sure to turn it off everywhere we need to, and that feels harder to maintain?

Reply:
I felt weird about doing it this way too, but it seemed like there was only one branch where it might ever turn off, so it'd end up more verbose. But if we both felt weird about it, might as well make it feel more normal. (Weird feels function as a DOS on the maintainer.)

Reply:
I don't feel *strongly* about it, if it really makes it that much more ugly. LMK.

Reply:
Made it smaller, even; go figure. :)